src/data/queries/mediaVideo.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryData, QueryFormatter, QueryParams } from 'next-drupal-query'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import { MediaVideo } from '@/types/formatted/media'
+import { DrupalMediaVideo } from '@/types/drupal/media'
+import { drupalClient } from '@/lib/drupal/drupalClient'
+
+// Define the query params for fetching node--media_video.
+export const params: QueryParams<null> = () => {
+  return new DrupalJsonApiParams().addInclude(['video'])
+}
+
+// Define the option types for the data loader.
+export type DataOpts = {
+  id: string
+  context?: string
+}
+
+// Implement the data loader.
+export const data: QueryData<DataOpts, DrupalMediaVideo> = async (
+  opts
+): Promise<DrupalMediaVideo> => {
+ const entity = await drupalClient.getResource<DrupalMediaVideo>(
+    'media--video',
+    opts.id,
+    {
+      params: params().getQueryObject(),
+    }
+  )
+
+  return entity
+}
+
+export const formatter: QueryFormatter<DrupalMediaVideo, MediaVideo> = (
+  entity: DrupalMediaVideo
+) => {
+  if(!entity) return null
+  return {
```

Comment:
I don't think these are the fields we'll want for videos.

Looking at https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/media/video/c737fbc5-9379-42ec-b5e6-1d75f3bab6bd?resourceVersion=id%3A187 the fields surfaced are:
```json
     "drupal_internal__mid": 187,
      "drupal_internal__vid": 187,
      "langcode": "en",
      "revision_created": "2019-06-14T13:58:10+00:00",
      "status": true,
      "name": "For Eiler tracking the status of his VA payments and claims is easy with eBenefits.  ",
      "created": "2019-06-14T13:58:06+00:00",
      "changed": "2019-06-20T20:36:47+00:00",
      "default_langcode": true,
      "revision_translation_affected": true,
      "metatag": [
        {
          "tag": "meta",
          "attributes": {
            "name": "title",
            "content": "| Veterans Affairs"
          }
        }
      ],
      "path": {
        "alias": null,
        "pid": null,
        "langcode": "en"
      },
      "content_translation_source": "und",
      "content_translation_outdated": false,
      "field_description": null,
      "field_duration": null,
      "field_publication_date": null,
      "field_media_in_library": true,
      "field_media_submission_guideline": null,
      "field_media_video_embed_field": "https://www.youtube.com/watch?v=L5aJqLS4ks4"
```

We'll want `name`, `field_description`, `field_duration`, `field_media_video_embed_field` and potentially others, but definitely those.